scripts/release.py

```diff
@@ -206,16 +206,237 @@ def create_draft_release(
+        }
+    }
+
+    notebook_json = json.dumps(interpolated_notebook, indent=4, sort_keys=True)
+
+    headers = {
+        "Accept": "application/json",
+        "Content-Type": "application/json",
+        "DD-API-KEY": dd_api_key,
+        "DD-APPLICATION-KEY": dd_app_key,
+    }
+    # create new release notebook
+    requests.post("https://api.datadoghq.com/api/v1/notebooks", data=notebook_json, headers=headers)
+
+    configuration = Configuration()
+    configuration.api_key["apiKeyAuth"] = dd_api_key
+    configuration.api_key["appKeyAuth"] = dd_app_key
+    with ApiClient(configuration) as api_client:
+        api_instance = NotebooksApi(api_client)
+        response = api_instance.list_notebooks(query=nb_name)
+
+    nb_id = response._data_store["data"][0]["id"]
+    nb_url = "https://ddstaging.datadoghq.com/notebook/%s" % (nb_id)
+
+    print("\nNotebook created at %s\n" % nb_url)
+
+    # eliminate duplicates of handles for the slack message
+    author_slack_handles = " ".join(set(author_slack_handles))
+
+    print("Message to post in #apm-python-release once deployed to staging:\n")
+    print(
+        """It's time to test the {version} release candidate! The owners of pull requests with release notes in version are {author_slack_handles}. 
+Everyone mentioned here: before the end of the day tomorrow, please ensure that you've filled in the testing strategy in the release notebook {nb_url} on all release notes you're the owner of, according to the expectations here. 
+Please also note when you expect the testing itself to be completed (ASAP preferred).
+You can start doing your tests immediately, using {version}. If you have questions or need help with anything, let me know! I'm here to help. Thanks all for your dedication to maintaining a rock-solid library.
+
+Check the release notebook {nb_url} for asynchronous updates on the release process. If you have questions or need help with anything, let me know! I'm here to help. Thanks all for your dedication to maintaining a rock-solid library.""".format(  # noqa
+            version=name, author_slack_handles=author_slack_handles, nb_url=nb_url
+        )
+    )
 
 
 if __name__ == "__main__":
-    create_release_draft()
+
+    subprocess.check_output(
+        "git stash",
+        shell=True,
+        cwd=os.pardir,
+    )
+    start_branch = (
+        subprocess.check_output(
+            "git rev-parse --abbrev-ref HEAD",
+            shell=True,
+            cwd=os.pardir,
+        )
+        .decode()
+        .strip()
+    )
+
+    #  Figure out the version of the library that you’re working on releasing grabbed with VERSION envar
+    base = os.getenv("BASE")
+    rc = bool(os.getenv("RC"))
+    patch = bool(os.getenv("PATCH"))
+
+    if base is None:
+        raise ValueError("Need to specify the base version with envar e.g. BASE=1.10")
+
+    dd_repo = setup_gh()
+    name, rn = create_release_draft(dd_repo, base, rc, patch)
+
+    if os.getenv("NOTEBOOK"):
```

Comment:
I think `argparse` would be better than all these env variables (IMHO)

Reply:
I think you're probably right, will add to backlog